src/fcc_bridge_node_mavsdk.cpp

```diff
@@ -0,0 +1,274 @@
+//
+// Created by Johan <job8197@thi.de> on 30.04.2024.
+//
+
+#include "fcc_bridge_node.hpp"
+
+// Libc header
+#include <cinttypes>
+#include <map>
+
+namespace fcc_bridge {
+namespace {
+// Path to uart device on Raspberry PI
+constexpr char const *UART_DEVICE_PATH = "/dev/serial0";
+
+// Baud rate to use for uart connection
+constexpr int UART_BAUD_RATE = 112500;
```

Comment:
This baud rate is not valid
```suggestion
constexpr int UART_BAUD_RATE = 115200;
```

Source: https://support.sbg-systems.com/sc/kb/latest/technology-insights/uart-baud-rate-and-output-rate#:~:text=19200%2C%2038400%2C%2057600%2C-,115200,-%2C%20230400%2C%20460800%2C%20921600

Reply:
I just saw that this was already fixed by @incrazyboyy

Reply:
yes, the baudrate used in the px4 docs is 57600